src/util/db.ts

```diff
@@ -48,11 +48,15 @@ export function createInstance(config: ConnectionConfig): Knex {
     throw new Error('The provided connection already contains a connection instance.');
   }
 
-  const { host, database } = config.connection as any;
+  const { host, server, database, userName } = config.connection as any;
```

Comment:
@kshitish182 This is slightly irritating. Can you check why config.connection has to be casted to any? We should avoid any.

Reply:
Yes, will look into it. We probably knex's type definition for this.